config/templates/Dockerfile

```diff
@@ -1,18 +1,28 @@
 FROM ubuntu:21.04
 ARG DEBIAN_FRONTEND=noninteractive
-RUN apt-get update && apt-get -y install joe software-properties-common
-RUN apt-get install -y --no-install-recommends \
+RUN apt-get update && apt-get -y install \
+       joe \
+       software-properties-common \
        gnupg \
        gnupg1 \
        gpgv1 \
     && rm -rf /var/lib/apt/lists/*
 RUN sh -c  "if [ x"" != x$http_proxy ]; then \
-			    apt-key adv --keyserver hkp://p80.pool.sks-keyservers.net:80 --keyserver-options http-proxy=$http_proxy --recv-keys ED75B5A4483DA07C >/dev/null 2>&1; \
-		    else \
-			    apt-key adv --keyserver hkp://p80.pool.sks-keyservers.net:80 --recv-keys ED75B5A4483DA07C >/dev/null 2>&1; \
-		    fi"
+                apt-key adv --keyserver hkp://p80.pool.sks-keyservers.net:80 --keyserver-options http-proxy=$http_proxy --recv-keys ED75B5A4483DA07C >/dev/null 2>&1; \
+            else \
+                apt-key adv --keyserver hkp://p80.pool.sks-keyservers.net:80 --recv-keys ED75B5A4483DA07C >/dev/null 2>&1; \
+            fi"
 RUN echo "deb http://repo.aptly.info/ nightly main" > /etc/apt/sources.list.d/aptly.list
```

Comment:
How to test `aptly` usage?
Is it used at all?

Reply:
It is used in the backstage to form and manipulate repository and for creating packages in the chroot.
https://github.com/armbian/build/blob/master/lib/chroot-buildpackages.sh#L329

If we move our main server to aarch64, then we would need this, otherwise not.